### DISCUSSION TITLE
Arri Alexa 35 metadata & Arri Embedded Look Appliance

### DISCUSSION THREAD

**Discussion by addit248** [videovillage/community#138] [Ideas]
Hi there,

The metadata for the Alexa 35 isn't supported right now. I'd have to check all my footage for metadata in the slow Arri Reference Tool. I'd love to also be able to change metadata or add metadata within Screen.

It would also be awesome apply the embedded look to the footage. You can even just check the name and check the folder for that name to apply. Keep in mind that the Alexa 35 takes ALF4 files and you have to use other look files (cube for example) for Screen. ALF4 would also be nice to add to Lattice.

Lattice would also need an update to make it possible to convert ALF2 (LOG to 709) to ALF4 (LOG to LOG), so it removes the ARRI709 conversion.

Thanks for reading!

**Comment by gregcotten** [comment]
Alexa 35 MXF ProRes should have the same shortcomings as an other Arri MXF ProRes, but I can't say for sure.

Re: AL4 - we will likely add rudimentary support for it soon, but we aren't super happy with ARRI's reinvention of the wheel.

Converting ALF2 to ALF4 is simply not possible without ARRI publishing how to reverse their forced ACES-like tone mapping+colorspace transformations required with ALF4. 

We are really hoping ARRI will come to their senses and add a creatives-friendly AL4 update that allows DPs and Colorists to have full control of their image pipeline. ARRI has somehow forgotten that display prep is indeed a very subjective part of the display prep look creation.
